group_vars/all

```diff
@@ -66,6 +66,10 @@
   login_node_prefix: "login-"
   num_login_nodes: 0
 
+#OpenOnDemand
+  ood: "ood"
+  ood_provision: true
+
```

Comment:
Got a warning while running 
ansible-playbook -c local -i /vagrant/CRI_XCBC/hosts -l `hostname` /vagrant/CRI_XCBC/ohpc.yaml --tags=pre_ohpc -s -vv

[WARNING]: Found both group and host with same name: ood